### DISCUSSION TITLE
WhatsApp Business

### DISCUSSION THREAD

**Discussion by tuxmartin** [42wim/matterbridge#1981] [New bridges suggestions]
Please add support for WhatsApp Business.

Integration https://github.com/42wim/matterbridge/wiki/Section-WhatsApp-%28basic%29 doesn't work with WhatsApp Business.

**Comment by divyaniverma** [comment]
WhatsApp Business is a messaging app specifically designed for businesses to communicate with their customers.It offers a verified Business Profile to enable greater trust in messaging support, which is suitable for managing lower volumes of customer requests for smaller businesses. Meanwhile, larger businesses with an enterprise, commercial.

Most businesses can use WhatsApp Business API to integrate the app into CRM and customer support software for greater scalability. WhatsApp Business offers many features that can benefit small businesses, including verified business profiles, quick replies, automated greetings, message statistics, interactive messages, integration with other platforms and more.

Read more msgclub.net
